app/assets/stylesheets/articles.scss

```diff
@@ -4,297 +4,74 @@
-        }
-        @media screen and (min-width: 650px) {
-          .narrow-nav-select {
-            display: none;
-          }
-          .wide-nav-links {
-            display: block;
-            cursor: default;
-          }
-        }
-        @media screen and (min-width: 950px) {
-          top: 9px;
-          &.sub-home-nav {
-            top: 8px;
-          }
-        }
-      }
-      .separator {
-        border-left: 2px solid $light-medium-gray;
-        display: inline-block;
-        margin: 0px 7px;
-        height: 15px;
-      }
-      .nav-chronofiter-link {
-        display: inline-block;
-        padding: 2px 8px;
-        font-size: 0.95em;
-        border-radius: 2px;
-        border: 1px solid transparent;
-        &.selected {
-          background: $purple;
-          color: $black;
-          border: 1px solid $dark-purple;
-          box-shadow: 3px 4px 0px $dark-purple;
-        }
-        &:hover {
-          opacity: 1;
-          background: $dark-purple;
-          color: white;
-          border: 1px solid $purple;
-          box-shadow: 3px 4px 0px $purple;
-        }
-      }
-      .narrow-nav-select {
-        button {
-          all: unset;
-          background: transparent;
-          border: 0;
-          font-family: $helvetica-condensed;
-          font-stretch: condensed;
-          font-weight: 600;
-          cursor: pointer;
-        }
-      }
+
+    @media screen and (min-width: 1120px) {
+      max-width: calc(100% - 280px - 280px);
     }
+    
     .pinned-articles {
-      @include themeable(background, theme-container-background, $white);
-      @include themeable(
-        border,
-        theme-container-border,
-        1px solid $outline-color
-      );
-      padding: 8px 0px;
-      margin: auto;
-      margin-top: 8px;
-      @media screen and (min-width: 750px) {
-        border-radius: 3px;
-        box-sizing: border-box;
-        padding: 8px 8px;
-        padding-right: 9px;
-        width: calc(100% + 2px);
-        max-width: 722px;
-      }
-      .pinned-articles-indicator {
-        text-align: left;
-        font-size: 15px;
-        font-weight: bold;
-        padding-top: 8px;
-        padding-left: 2.8%;
-        @media screen and (min-width: 750px) {
-          padding-left: 22px;
-        }
-        @media screen and (min-width: 950px) {
-          padding-left: 2px;
-        }
-        img {
-          width: 20px;
-          transform: rotate(-8deg);
-          vertical-align: -5px;
-          @include themeable(filter, theme-social-icon-invert, invert(0));
-        }
+      padding: 8px;
+      margin: 0 auto;
+      @include dev-card;
+
+      h5 {
+        margin: 0 0 8px;
+        padding: 8px 0;
+        display: flex;
+        align-items: center;
       }
     }
+
     a {
       @include themeable(color, theme-container-color, $black);
     }
-    a.small-pic-link-wrapper,
-    .placeholder-div {
+
+    .placeholder-div { // TOOD: what is this?
```

Comment:
There is an empty `<div />` for related to this class in a few files, but it's a `<div />`, so `display: block` is redundant as that's the default for a `<div />` and even the width is redundant as block elements take up the whole width by default.

![image](https://user-images.githubusercontent.com/833231/74498078-77890a80-4ead-11ea-89fa-0a827f79fb2c.png)

Reply:
Yes, I was looking for that as well. I found commit form several months ago adding this thing. You're right CSS doesn't make much sense here but so do the element in DOM.  So I'd rather dig a bit more and make sure I can remove both: html & css.